Octokit/Models/Response/CommitActivity.cs

```diff
@@ -13,13 +14,15 @@ public class CommitActivity
 
         public CommitActivity(IEnumerable<WeeklyCommitActivity> activity)
         {
-            Activity = activity;
+            Ensure.ArgumentNotNull(activity, "activity");
```

Comment:
I really want to get Fody.NullGuard integrated into this. So I do care, but not that much.